jwst/resample/resample.py

```diff
@@ -545,4 +547,4 @@ def update_slit_metadata(self, model):
                 pass
             else:
                 if val is not None:
-                    setattr(model, attr, val)
+                    setattr(model, attr, val)
```

Comment:
Probably need to remove the blank line after this, in order to keep flake8 happy.